src/main/java/TaskList.java

```diff
@@ -0,0 +1,67 @@
+import java.util.ArrayList;
+
+public class TaskList {
+    private ArrayList<Task> list = new ArrayList<Task>();
+    private int numberOfTasks = 0;
+    private int numberUndone = 0;
+    private int numberDone = 0;
+
+
+
+    public TaskList() {
+
+    }
+
+    public void addTask(Task task) {
+        this.list.add(task);
+        System.out.println("Got it! I have added the following task: \n    " + task.toString());
+        numberUndone++;
+        numberOfTasks++;
+    }
+
```

Comment:
missing header comments for **all** public methods. do take note of the same mistakes in other files as well